src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+- ../System/System
+connector:
+  displayName: LinuxProcess
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
+    - type: commandLine
+      commandLine: /usr/bin/uname -o || /bin/uname -o
+      expectedResult: GNU/Linux
+      errorMessage: Not a valid Linux host.
+    tags: [ system, linux ]
+monitors:
+  process:
+    simple:
+      sources:
+        process:
+          type: commandLine
+          commandLine: ps aux | grep '${var::processNames}'
+          computes:
+          - type: awk
+            script: |
+              {
+                pid = $2
+                cpu = $3
+                mem = $4
+                command = $11
+
+                # Converting time from HH:MM to seconds
```

Comment:
The format is not always HH:MM, IIRC, especially for processes running for a long time. But there are options to display the time in seconds. Check the ps man page.